actions/internal/plugins/package/package.sh

```diff
@@ -110,6 +110,8 @@ for file in $(find "$backend_folder" -type f -name "${exe_basename}_*"); do
     # Copy all files but the executables
     mkdir -p "$plugin_id"
     rsync -a --exclude "${exe_basename}*" "$dist/" "$plugin_id"
+    # TODO: this instead of rsync
+    # find "$dist" -type f ! -name "${exe_basename}*" -exec cp --parents {} "$plugin_id" \;
```

Comment:
Related to act testing: the Docker image used for act doesn't include `rsync` unlike GitHub Actions. We currently install it via `apt` in the setup step, but it's an unnecesary step and a waste of time. We can use `find` instead of `rsync`, which is already installed. Just adding a comment for now, but I will add this later on as an improvement